addons/product/wizard/update_product_attribute_value.py

```diff
@@ -25,7 +25,7 @@ def _compute_message(self):
         for wizard in self:
             if wizard.mode == 'add':
                 wizard.message = _(
-                    "You are about to add \"%(attribute_value)s\" to %(product_count)s products.",
+                    "You are about to add \"%(attribute_value)s\" attribute value to %(product_count)s products.",
```

Comment:
You need to update the pot for such a change

Reply:
```suggestion
                    "You are about to add the value \"%(attribute_value)s\" to %(product_count)s products.",
```

Reply:
even if you rebase your branch ?
if it's not in the POT at all you don't have to do anything, but if it's there, you should update it.

Reply:
Ok then, maybe they still have to regenerate it :D